rpcs3/Input/basic_mouse_handler.cpp

```diff
@@ -89,11 +89,23 @@ void basic_mouse_handler::MouseScroll(QWheelEvent* event)
 	MouseHandlerBase::Scroll(event->angleDelta().y());
 }
 
+bool basic_mouse_handler::getMouseLockState()
```

Comment:
I don't like that this function is duplicated.
There surely is a way to make this accessible in both classes without implementing it twice

Reply:
As it stands, keyboard_pad handler only inherits from QObject, PadHandlerBase.  Whilst mouse handler inherits from QObject, MouseHandlerBase.
Perhaps keyboard_pad handler should have inherited from MouseHandlerBase also, especially given they both capture and handle Qt Mouse Events.
Is there a reason this wasn't already the case? (quite a lot of their MouseEvent functions are very similar, but slightly different)

Reply:
Because one is a pad handler and the other is a mouse handler, which both have nothing to to with each other